job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext/buildResult.groovy

```diff
@@ -0,0 +1,25 @@
+job('example') {
+    triggers {
+        buildResult('H/10 * * * *') {
+            combinedJobs()
+            triggerInfo('My job 1') {
```

Comment:
My first approch is like you say, but I change my implementation for respect mandatory rules from the plugin.